package.json

```
@@ -35,12 +35,14 @@
     "mqtt": "^5.10.3"
   },
   "devDependencies": {
-    "ajv": "8.17.1",
+    "@eslint/js": "^9.36.0",
     "aedes": "^0.49.0",
-    "eslint": "^8.48.0",
-    "eslint-config-standard": "^17.0.0",
+    "ajv": "8.17.1",
+    "eslint": "^9.36.0",
```

```suggestion
    "eslint": "^9.39.2",
```

any reason not to go with latest?